server/models/center.js

```diff
@@ -0,0 +1,34 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const Center = sequelize.define('Center', {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+  },
+  location: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false
+  },
+  description: {
+      type: DataTypes.STRING,
+      allowNull: false
+  },
+  facilities: {
+    type: DataTypes.STRING,
+    unique: true,
+    allowNull: false
+},
+  }, {
+    classMethods: {
```

Comment:
I don't think class methods work for `pg version` 7 upwards. Read the docs for clarity.

Reply:
ok. thanks